### ISSUE TITLE
[FEATURE] Warn when the library packages the test folder as a module

### ISSUE THREAD

**Issue by ocefpaf** [conda-incubator/grayskull#268]
Many packages, `xyzservices`, wrongfully packages the test folder as a module. If one tries:

```
grayskull pypi --strict-conda-forge xyzservices
```

we will see that grayskull correctly identify this issue by adding the test module in the import section:

```yaml
test:
  imports:
    - tests
    - xyzservices.providers
    - xyzservices
```

We should use that in our advantage and warn users that this package does the "wrong" thing and will cause clobber errors with others that do the same. The reason we need a warning is b/c most of us got the habit of just deleting that line and ignoring the issue until it blows up in our face with a CondaClobberWarning :-)

**Comment by BastianZim** [comment]
Have also seen that with `examples`. But how about adding this to the linter instead?

**Comment by marcelotrevisani** [comment]
Fixed with #280 and #279

Related to #262
